users/views.py

```diff
@@ -1,3 +1,55 @@
-from django.shortcuts import render
+import json, jwt, requests
+
+
+from django.views import View
+from django.http  import JsonResponse
+
+from users.models import User, UserLevel
+from my_settings  import SECRET_KEY, REST_API_KEY
+
+class KaKaoSignInView(View):
+    def post(self, request):
+        try:
+            REDIRECT_URI = "http://localhost:3000/users/kakao"
+            POINT        = 1000000
+            
+            data = json.loads(request.body)
+            code = data['code']
+
+            kakao_token_request = requests.get(    
+                f"https://kauth.kakao.com/oauth/token?grant_type=authorization_code&client_id={REST_API_KEY}&redirect_uri={REDIRECT_URI}&code={code}",
+            )
+            kakao_json = kakao_token_request.json()
+            profile_request = requests.get(    
+                "https://kapi.kakao.com/v2/user/me",
+                headers = {"Authorization" : f"Bearer {kakao_json['access_token']}"}
+            )
```

Comment:
requests 모듈을 사용하실 때에는 timeout 옵션을 반드시 설정해주셔야 합니다.
그 이유는 timeout option을 추가하지 않으면 서버의 응답을 계속해서 기다리고 있는 문제가 생깁니다.

`timeout` 옵션을 추가해보세요~
```suggestion
            profile_request = requests.get(    
                "https://kapi.kakao.com/v2/user/me",
                headers = {"Authorization" : f"Bearer {kakao_json['access_token']}"}
            )
```